### DISCUSSION TITLE
Is there a way to access monthly statements via the shop manager?

### DISCUSSION THREAD

**Discussion by totokpeter** [etsy/open-api#747] [Q&A] [answered]
I was unable to find anything regarding monthly statements. I am trying to automate statement distribution to my accountant and looking for ways. Thanks in advance!

**Comment by etsyachristensen** [comment, marked as answer]
I'm sorry but we can't really help you with Shop Manager questions. You will have to go through the Help Center for that: https://help.etsy.com/hc/en-us?segment=shopping

You could create an app and query your receipts by the start/end of the month in the API call here: https://developers.etsy.com/documentation/reference/#operation/getShopReceipts

**Reply by totokpeter** [reply]
Thank you! I was hoping to get some idea of what can and cannot be done with the API and I am grateful for the prompt answer!
